src/Workspaces/Core/Portable/Storage/StorageOptions.cs

```diff
@@ -17,8 +17,15 @@ internal static class StorageOptions
 
         public const string OptionName = "FeatureManager/Storage";
 
-        public static readonly Option<StorageDatabase> Database = new(
-            OptionName, nameof(Database), defaultValue: StorageDatabase.SQLite);
+        public static readonly Option<StorageDatabase> Database = new(OptionName, nameof(Database), defaultValue: StorageDatabase.CloudCache);
```

Comment:
❔ Shouldn't we leave SQLite as the default for now?

Reply:
yes.  thais was for my testing :)